### PR TITLE
feat: network, market, and icon config for Metis Andromeda [skip cypress]

### PR DESCRIPTION
**Pull request by defispartan**
## General Changes

From governance proposal https://app.aave.com/governance/proposal/?proposalId=212

## Developer Notes

Add any notes here that may be helpful for reviewers.

---

## Author Checklist

Please ensure you, the author, have gone through this checklist to ensure there is an efficient workflow for the reviewers.

- [ ]  The base branch is set to `main`
- [ ]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [ ]  The Github issue has been linked to the PR in the Development section
- [ ]  The General Changes section has been filled out
- [ ]  Developer Notes have been added (optional)

**If the PR is ready for review:**

- [ ]  The PR is in `Open` state and not in `Draft` mode
- [ ]  The `Ready for Dev Review` label has been added

## Reviewer Checklist

Please ensure you, as the reviewer(s), have gone through this checklist to ensure that the code changes are ready to ship safely and to help mitigate any downstream issues that may occur.

- [ ]  End-to-end tests are passing without any errors
- [ ]  Code style generally follows existing patterns
- [ ]  Code changes do not significantly increase the application bundle size
- [ ]  If there are new 3rd-party packages, they do not introduce potential security threats
- [ ]  If there are new environment variables being added, they have been added to the `.env.example` file as well as the pertinant `.github/actions/*` files
- [ ]  There are no CI changes, or they have been approved by the DevOps and Engineering team(s)
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [ ]  Squash and merge has been checked

### CODE REVIEW

File: src/ui-config/networksConfig.ts [hunk 1]
```diff
@@ -381,4 +381,14 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
       url: 'https://app.multichain.org/#/router',
     },
   },
+  [ChainId.metis_andromeda]: {
+    name: 'Metis Andromeda',
+    privateJsonRPCUrl: 'https://metis-mainnet.gateway.pokt.network/v1/lb/62b3314e123e6f00397f19ca',
+    publicJsonRPCUrl: ['https://andromeda.metis.io/?owner=1088'],
+    baseAssetSymbol: 'ETH',
+    wrappedBaseAssetSymbol: 'WETH',
+    baseAssetDecimals: 18,
```
Comment (brotherlymite) [inline src/ui-config/networksConfig.ts:390]:
```suggestion

```

File: src/ui-config/networksConfig.ts [hunk 2]
```diff
@@ -381,4 +381,14 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
...
+    explorerLink: 'https://andromeda-explorer.metis.io/',
```
Comment (brotherlymite) [inline src/ui-config/networksConfig.ts:391]:
```suggestion
    explorerLink: 'https://andromeda-explorer.metis.io',
```

**Comment by brotherlymite** [comment]
@defispartan to fix the icons for the listed asset, probably we can add this:
```
  // metis
  'm.USDC': 'USDC',
  'm.USDT': 'USDT',
  'm.DAI': 'DAI',
```
 to https://github.com/aave/interface/blob/feat/metis-config/src/ui-config/reservePatches.ts#L40

**Comment by brotherlymite** [comment]
@defispartan this script might be helpful to run the anvil fork locally with executed payload to test stuff on the interface
https://github.com/bgd-labs/aave-v3-metis-proposal/tree/feat/v3-metis-payload#anvil-fork-simulation

File: src/ui-config/networksConfig.ts (hunk 1 above)
Comment (defispartan) [reply src/ui-config/networksConfig.ts]: Removing completely breaks some types across the app, but leaving them as blank will remove the base asset from the "Assets To Supply" which I think is what's intended here

**Review by grothem — APPROVED** [review]

**Pull request merged**